packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.ts

```diff
@@ -34,169 +36,205 @@ import { data } from './data';
-
-  public searchText = '';
-  public caseSensitive = false;
-
-  public positionSettings: PositionSettings = {
-    horizontalDirection: HorizontalAlignment.Left,
-    horizontalStartPoint: HorizontalAlignment.Right,
-    verticalStartPoint: VerticalAlignment.Bottom
-  };
-
-  public overlaySettings: OverlaySettings = {
-    closeOnOutsideClick: true,
-    modal: false,
-    positionStrategy: new ConnectedPositioningStrategy(this.positionSettings),
-    scrollStrategy: new CloseScrollStrategy()
-  };
-
-  constructor(private crd: ChangeDetectorRef, private excelExporterService: IgxExcelExporterService) { }
-
-  public ngOnInit() {
-    this.localData = data;
-  }
-
-  public toggleHiding() {
-    this.overlaySettings.positionStrategy.settings.target = this.hidingButton.nativeElement;
-    this.toggleRefHiding.toggle(this.overlaySettings);
-  }
-
-  public togglePinning() {
-    this.overlaySettings.positionStrategy.settings.target = this.pinningButton.nativeElement;
-    this.toggleRefPinning.toggle(this.overlaySettings);
-  }
-
-  public ngAfterViewInit() {
-    this.grid1.summariesHeight = 60;
-    this.grid1.reflow();
-    this.cols = this.grid1.columnList;
-    this.hiddenColsLength = this.cols.filter((col) => col.hidden).length;
-    this.pinnedColsLength = this.cols.filter((col) => col.pinned).length;
-    this.crd.detectChanges();
-  }
-
-  public toggleVisibility(col: IgxColumnComponent) {
-    if (col.hidden) {
-      this.hiddenColsLength--;
-    } else {
-      this.hiddenColsLength++;
-    }
-    col.hidden = !col.hidden;
-  }
-
-  public togglePin(col: IgxColumnComponent, evt) {
-    if (col.pinned) {
-      this.grid1.unpinColumn(col.field);
-      this.pinnedColsLength--;
-    } else {
-      if (this.grid1.pinColumn(col.field)) {
-        this.pinnedColsLength++;
-      } else {
-        // if pinning fails uncheck the checkbox
-        evt.checkbox.checked = false;
-      }
-    }
-  }
-
-  public exportData() {
-    this.excelExporterService.exportData(this.localData, new IgxExcelExporterOptions('Report'));
-  }
-
-  public formatDate(val: Date) {
-    return new Intl.DateTimeFormat('en-US').format(val);
-  }
-
-  public searchKeyDown(ev) {
-    if (ev.key === 'Enter' || ev.key === 'ArrowDown' || ev.key === 'ArrowRight') {
-      ev.preventDefault();
-      this.grid1.findNext(this.searchText, this.caseSensitive);
-    } else if (ev.key === 'ArrowUp' || ev.key === 'ArrowLeft') {
-      ev.preventDefault();
-      this.grid1.findPrev(this.searchText, this.caseSensitive);
-    }
-  }
-
-  public updateSearch() {
-    this.caseSensitive = !this.caseSensitive;
-    this.grid1.findNext(this.searchText, this.caseSensitive);
-  }
-
-  public clearSearch() {
-    this.searchText = '';
-    this.grid1.clearSearch();
-  }
-
-  public formatValue(val: any): string {
-    return val.toLocaleString('en-us', { maximumFractionDigits: 2 });
-  }
+	@ViewChild("grid1", { read: IgxGridComponent, static: true })
```

Comment:
A bit nit-picky, but the indentation here is incorrect.
Also, all of there are several other lint errors:
 - Double quotation marks `"` should be single `'`
 - Underscored properties can be renamed so that lint doesn't get upset